README.md

```diff
@@ -1 +1,135 @@
-# jx-app-jacoco
\ No newline at end of file
+# jx-app-jacoco
+
+jx-app-jacoco provides a means for transferring a Jacoco XML code coverage report 
+from a [Jenkins X](https://jenkins-x.github.io/jenkins-x-website/) build to 
+a fact in the PipelineActivity CRD.
+
+You must have a Jenkins X cluster to install and use the jx-app-jacoco app.  If
+you do not have a Jenkins X cluster and you would like to try it out, the 
+[Jenkins X Google Cloud Tutorials](https://jenkins-x.io/getting-started/tutorials/) 
+is a great place to start.
+
+## Installation
+
+Using the [jx command line tool](https://jenkins-x.io/getting-started/install/), run 
+the following command.
+
+`jx add app -n "" jx-app-jacoco --repository "http://chartmuseum.jenkins-x.io"`
```

Comment:
-n ""? Empty quotes?

Reply:
Bug.  The namespace field has to be there for the moment.

Reply:
Pete's CR to address this in the works - https://github.com/jenkins-x/jx/pull/2945